tests/unit/test_run.py

```diff
@@ -2295,6 +2295,23 @@ def test_interpolate(combo_df):
     npt.assert_array_almost_equal(res.values.squeeze(), combo.target_values)
 
 
+def test_interpolate_float(scm_run):
+    target_time_points = [float(y) for y in range(2000, 2005)]
+
+    res = scm_run.interpolate(target_time_points)
+    assert res["year"].to_list() == [2000, 2001, 2002, 2003, 2004]
+    assert np.isnan(res.values).sum() == 0
+
+
+def test_interpolate_int(scm_run):
+    # check int not converted to unix time (gh 221)
+    target_time_points = [int(y) for y in range(2000, 2005)]
+
+    res = scm_run.interpolate(target_time_points)
+    assert res["year"].to_list() == [2000, 2001, 2002, 2003, 2004]
+    assert np.isnan(res.values).sum() == 0
```

Comment:
Does this fail if the int is converted to unix time?

Reply:
No. The interpolation would work as expected, in that you would get non-nan values, but the results can be very wrong If linear extrapolation was used.

Reply:
You would get non-nan values but they would be for time points in 1970 so completely wrong? (I'm just trying to get my head around how you arrive at "The interpolation would work as expected", I'm assuming that should instead be, "The interpolation would work i.e. you would get non-nan values but because it is interpolating for values in 1970 it is very different from what you want if you're assuming that integers are assumed to be years)"

Reply:
If this line doesn't tell us anything about whether the unix time is handled correctly, I would delete it or change it.

Suggestions for how to fix it:

- assert the actual values (we should be able to check them fairly easily and they would come out differently if we were accidentally interpolating for 1970 instead?)
- set extrapolation equal to `None` or something, so the values come out as `np.nan` if the int is interpolated as unix time